xtask/src/ci/target_matrix.rs

```diff
@@ -11,6 +11,24 @@ pub(crate) fn run(message: String, author: String) -> Result<(), color_eyre::Rep
     let mut matrix: Vec<CiTarget> = get_matrix().clone();
     let (prs, mut app) = if author == "bors[bot]" {
         process_bors_message(&message)?
+    } else if author == "[gha]" {
```

Comment:
im not sure this is the best approach

Reply:
Why not just use an environment variable here? We don't need to hack the author if we set an envvar upon this workflow, or a CLI argument. Something like:

```rust
    TargetMatrix {
        #[clap(long, env = "COMMIT_MESSAGE")]
        message: String,
        #[clap(long, env = "COMMIT_AUTHOR")]
        author: String,
        // if the check is being run as part of a weekly check
        #[clap(long, env = "WEEKLY_CI", value_parser = BoolishValueParser::new())]
        weekly: Option<bool>,
    },
```